examples/pio_blink/rpx0xx_pio_blink/blink.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2023 Fabian Hüßler
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <errno.h>
+#include "pio/pio.h"
+#include "blink.pio.h"
```

Comment:
What is this file ? It doesn't seems to exist in the tree.

Reply:
```C
#include "pio/pio.h" /* this is the internal PIO header of the rpx0xx */
#include "blink.pio.h" /* this is the auto generated header from blink.pio */
```

Is the last one not generated for you or did you not compile the blink example?
The .pio.h are not supposed to be pushed because they are generated.

Reply:
Oh I didn't compile the example before checking this.
Indeed the file is generated after the compilation !